glue/viewers/image/qt/slice_widget.py

```diff
@@ -142,7 +159,7 @@ class FakeViewerState(State):
     viewer_state.y_att = data.pixel_component_ids[3]
     viewer_state.slices = [0] * 5
 
-    widget = MultiSliceWidgetHelper(viewer_state)
+    widget = ImageMultiSliceWidgetHelper(viewer_state)
```

Comment:
If possible it would be good to avoid the change in the class name unless absolutely needed - since this is public API that may be used by other plugins.

Reply:
Yes, agreed. Let me revert this.